test/remote/gateways/remote_cyber_source_rest_test.rb

```diff
@@ -70,6 +70,27 @@ def test_successful_purchase
     assert_nil response.params['_links']['capture']
   end
 
+  def test_successful_refund
+    purchase = @gateway.purchase(@amount, @visa_card, @options)
+    response = @gateway.refund(@amount, { order_id: purchase.authorization })
+
+    assert_success response
+    assert response.test?
+    assert_equal 'PENDING', response.message
+    assert response.params['id'].present?
+    assert_nil response.params['_links']['capture']
+  end
+
+  def test_successful_credit
+    response = @gateway.credit(@amount, @visa_card, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'PENDING', response.message
+    assert response.params['id'].present?
+    assert_nil response.params['_links']['capture']
+  end
```

Comment:
💬 Comment:

Maybe is a good idea to test not just the happy path:

- maybe checking how to handle error handling in case of a double refund
- doing partial refunds and values above the initial transaction
- credits on invalid credit cards
- etc.

Reply:
Done!